posthog/api/insight.py

```diff
@@ -1090,6 +1090,7 @@ def activity(self, request: request.Request, **kwargs):
         return activity_page_response(activity_page, limit, page, request)
 
     @action(methods=["POST"], detail=False)
+    @monitor(feature=Feature.INSIGHT, endpoint="insight", method="CANCEL")
```

Comment:
Adding monitoring here as we flood kill requests it can max out simultaneous queries.
We should look into if we should still be sending KILL requests to clickhouse.